docs/src/branching_releases.md

```diff
@@ -0,0 +1,76 @@
+
+With our own process we hope to achieve a structured release cycle that is consistent, and dependable. For both users of the protocol, and developers of the protocol to have understandable expectations about when and how releases will happen. And what can be expected within each release.
+
+Predominantly the team at Tari uses the Gitflow² branching model for work on the protocol. We won't linger long over the description of this flow unless necessary, but you should consider yourself acquainted with the process before continuing on. 
+
+## Channels
+
+We run a series of networks for different purposes but each network will have come from one of these three channels.
+
+- TestNet
+- NextNet
+- StageNet/MainNet
+
+### Networks
+
+- MainNet: The big shebang, the head hancho, the real deal. This is the Tari protocol at work.
+- StageNet: A stable network as similar as possible to MainNet available for testing.
+- NextNet: The next realease of the network. This should become available 8 weeks prior to going live on MainNet.
+- TestNet(s): Our named test networks used for development. Where we can play a little fast and loose with features where, although we try to avoid it- the networks may see breakages and resets more frequently.
+  - Example networks (non exhaustive):
+    - `Esme`: Has been our stable-er testnet prior to StageNet. It sees new features and enhancements as they come out, and supports our mobile wallets Tari Aurora on iOS and Android.
+    - `Igor`: Is used in the development of the DAN Layer and sees frequent resets to help support the rapid pace of the development team.
+
+## Releases
+
+Let's define some facts for our discussion about releasing.
+
+- StageNet/MainNet is at v1.53.0
+- NextNet is at v1.54.0-rc.0
+- TestNet is at v1.55.0-pre.4
+
+The naming practice here helps to differentiate where any particular version is meant to be deployed too. Any version containing `pre` is destined for a TestNet and will have all features enabled. A version containing `rc` is destined for NextNet as it is the release candidate for the future StageNet/MainNet release. Any singular version with no pre-release version appended such as v1.53.0 means this is a StageNet/MainNet build. No in development features should be enabled when this is compiled.
+
+### Development releases
+
+A developer writes a new feature `Feature-Y`. It is written behind a feature gate. They merge this new feature into `development`. They now want to run this feature on the `Esme` test network. They create a disposable release branch in the name of the version that will be tagged with the preface `testnet` and incrementing the number at the end of the `pre` version `testnet-1.55-pre.5`. Once any upgrade related changes are made such as incrementing the version through different crates, the developer will then tag the version `v1.55.0-pre.5`. The branch can be merged back into development, and the branch deleted.
```

Comment:
This paragraph isn't terribly clear.

esp this clause, "a disposable release branch in the name of the version that will be tagged with the preface `testnet`"

Could you try rewriting the paragraph?

Reply:
Yup will do. It needs some work.